docs/using-radicle/creating-patches.md

```diff
@@ -0,0 +1,80 @@
+---
+id: creating-patches
+title: Creating patches
+---
+
+To contribute to another user's project, you must first find it on the network.
+To find a project, you need its [**Radicle ID**][ri]. This can be found in the
+project page under the project name, or in a seed node dashboard such as
+[sprout.radicle.xyz](http://sprout.radicle.xyz).
+
+![Identity][id]
+
+Once you have the Radicle ID, copy it into the Upstream search bar. From there,
+you can follow the project to track it. This will replicate it to your local
+machine once it's found in your network of connected peers. You won't see any
+information on the project until it is verified by the protocol.
+
+![Search bar][sb]
+
+![Found Project][fp]
+
+### Forking projects
+
+To create your own view of a project, you can use the **Fork** option within the
+Upstream client. This will clone the project to a specified location on your
+local machine and publish your version to the Radicle network as a new project.
+
+![Fork Project][fo]
+
+Forking a project will move it from the **Following** tab to the **Projects**
+tab. This is because once you fork a project, you create your own view of it
+that is unique to your Device ID. You can [push changes][pc] to this view via
+the `rad` remote and publish them to the network. You are able to clone your
+project with the **Checkout** option on your project page.
+
+### Creating a patch
+
+Now that you have your own view of the project and you have it checked out, you
+can start creating a patch. In your view of the project you'll see a **Patches**
+tab next to **Files** and **Commits**. There you see a **New patch** button
+which will guide you through the steps.
+
+![New Patch][np]
+
+First, make your changes and commit them locally. Next, package them as a patch
+by creating an annotated git tag:
+
+`git tag --annotate radicle-patch/<name>`
+
+Be sure to replace `<name>` with the name of your patch.
+
+Next git will open your editor and allow you to add a title and description (the
+first line is the title, the second line is the description). Be sure to add
```

Comment:
Usually (as with Git commit messages) the description is separated by an empty line from the title. This is something we should describe properly. We should also clarify that the description can be multiple lines.


~~~suggestion
Next git will open your editor and allow you to add a title and description like this:

```plain
My patch Title

some more information
on multiple lines.
```
~~~